test/machineslib.py

```diff
@@ -212,7 +212,7 @@ def deleteIface(self, iface):
         b = self.browser
 
         b.click(f"#delete-vm-subVmTest1-iface-{iface}")
-        b.wait_in_text(".pf-c-modal-box .pf-c-modal-box__header .pf-c-modal-box__title", "Delete network interface")
+        b.wait_in_text(".pf-c-modal-box .pf-c-modal-box__header .pf-c-modal-box__title", "Delete network interface?")
```

Comment:
Please add a check for the description details in the dialog.

Reply:
Virtual network static DHCP entries doesn't have any object description in the dialog, it only a sentence "network {networkMac} will be permanently deleted.". So the only thing which could go wrong is to have an incorrect network mac. But we test in other places that we have correct MAC for a network entry, so is it worth testing?

Reply:
Actually, on second thought, we can never now what can go wrong, so let's lest, it. Updated the PR